packages/compass-query-bar/src/components/legacy-query-option/query-option.module.less

```diff
@@ -45,7 +45,8 @@
   margin-right: 5px;
   text-transform: uppercase;
   font-weight: bold;
-  font-family: 'Akzidenz', 'Helvetica Neue', Helvetica, Arial, sans-serif;
+  font-family: 'Euclid Circular A', 'Akzidenz', 'Helvetica Neue', Helvetica,
```

Comment:
Any reason we are still keeping the Akzidenz around?

Reply:
Nah, wasn't intended